lib/miq_environment.rb

```diff
@@ -14,6 +14,11 @@ def self.supports_apache?
       @supports_apache = self.is_appliance? && self.supports_command?('apachectl')
     end
 
+    def self.supports_systemd?
+      return @supports_systemd unless @supports_systemd.nil?
+      @supports_systemd = self.is_appliance? && self.supports_command?('systemctl')
```

Comment:
The `self.` bits are not needed here.

Reply:
It is consistent with the rest of the class, I'll put a PR in to clean this up so we're not refactoring in this PR